src/analytics/private/sqls/redshift/session-m-view.sql

```diff
@@ -0,0 +1,36 @@
+CREATE MATERIALIZED VIEW {{schema}}.session_m_view 
```

Comment:
If the event time span is large, the data volume of the Session will be quite huge. "row_number = 1" will prevent this MV from being incrementally refreshed.